mods/d2k/languages/rules/en.ftl

```diff
@@ -0,0 +1,18 @@
+## Player
+low = Low
+medium = Medium
+no-powers = No Powers
+unrestricted = Unrestricted
+
+automatic-concrete =
+    .label = Automatic Concrete
+    .description = Concrete foundations are automatically created beneath buildings
+
+## World
+map-creeps =
+    .label = Worms
+    .description = Worms roam the map and devour unprepared forces
```

Comment:
Is this the correct way of doing this? We should just use `map-worms` name instead

Reply:
I don't think that's strictly necessary. We also use the `MapCreeps` trait in D2k, not a version renamed to `MapWorms`. ;)

Reply:
It is necessary if you want to be consistent with every other translation